downstream-sample/README.md

```diff
@@ -0,0 +1,27 @@
+# Downstream delivery sample
+
+This service is an example of listening to `article` events to deliver new articles to downstream services such as [Crossref](https://www.crossref.org/).
+
+## Features
+
+Being a sample, this service does not interact with the real crossref service.
+
+Implemented:
+
+- emit an `article-delivery` event upon successful receival of the `article` event
```

Comment:
We're going to need to match these up between the Article Store, Dashboard and this.

The Dashboard currently knows `article.published`: https://github.com/libero/walking-skeleton/blob/9bb31a58dd2c783521d8f15dffa47b9ccecdff82/dashboard/translations/messages.en.yaml#L9-L10

(It'll display the raw name otherwise.)

Maybe this should listen for `article.published`, emit `article.crossref.started` and later `article.crossref.finished`?

Reply:
Just seen https://github.com/libero/walking-skeleton/blob/89b9fee5df3e2e85c76aa5b34057e89131f4df74/downstream-sample/downstream_sample/settings.py#L17

This is https://github.com/libero/walking-skeleton/pull/28#discussion_r189177476.

Reply:
Let's move there, general discussion